mdbenchmark/mdengines/gromacs.py

```diff
@@ -35,10 +37,23 @@ def prepare_benchmark(name, relative_path, *args, **kwargs):
 
     filepath = os.path.join(relative_path, full_filename)
 
-    copyfile(filepath, benchmark[full_filename].relpath)
+    if kwargs["multidir"] == 1:
+        copyfile(filepath, benchmark[full_filename].relpath)
+    else:
+        for i in range(kwargs["multidir"]):
+            subdir = benchmark[MULTIDIRS[i]+"/"+full_filename].make()
+            copyfile(filepath, subdir.relpath)
 
     return name
 
+def prepare_multidir(multidir):
+    if multidir == 1:
+        return ""
+    else:
+        multidir_string = "-multidir"
+        for i in range(multidir):
+            multidir_string += " "+MULTIDIRS[i]
+        return multidir_string
```

Comment:
We can rearrange this code to only return once.

```suggestion
    multidir_string = ""
    
    if multidir != 1:
        multidir_string = "-multidir"
        for i in range(multidir):
            multidir_string += " " + LOWERCASE_LETTERS[i]
            
    return multidir_string
```

But what I do not understand, why are we defining `multidir_string = "-multidir"`, instead of `"_multidir"`?

Reply:
yes that is much better!

It is "-multidir" because that is being inserted in the template, so it is the gromacs option.

Reply:
Now I understand, OK!